crates/bevy_winit/src/lib.rs

```diff
@@ -230,6 +230,10 @@ fn change_window(
                     let window = winit_windows.get_window(id).unwrap();
                     window.set_always_on_top(always_on_top);
                 }
+                bevy_window::WindowCommand::SetCursorHitTest { hittest } => {
+                    let window = winit_windows.get_window(id).unwrap();
+                    window.set_cursor_hittest(!hittest).unwrap();
```

Comment:
```suggestion
                    window.set_cursor_hittest(hittest).unwrap();
```
why the negation here?